patroni/api.py

```diff
@@ -547,3 +547,8 @@ def reload_config(self, config):
                 and self.__initialize(config):
             self.start()
         self.__set_config_parameters(config)
+
+    @staticmethod
+    def handle_error(request, client_address):
+        import traceback
+        logger.debug(traceback.format_exc())
```

Comment:
I think it should be at least "info" or "warning" and if you really to avoid spamming in the logs, you can change a log level for `patroni.api` to 'WARNING' or 'ERROR', https://github.com/zalando/patroni/pull/927 adds support for that.

Reply:
I've added `client_address` into logs and used `warning` level for this function.